malduck/procmem/procmem.py

```diff
@@ -11,7 +11,19 @@
 
 __all__ = ["ProcessMemory", "procmem"]
 
-ProcessMemoryBuffer = Union[bytes, bytearray, mmap.mmap]
+
+class MemoryBuffer(object):
+    def __setitem__(self, item, value):
+        raise NotImplementedError("__setitem__ not implemented")
+
+    def __getitem__(self, item):
+        raise NotImplementedError("__getitem__ not implemented")
+
+    def __len__(self):
+        raise NotImplementedError("__len__ not implemented")
+
+
+ProcessMemoryBuffer = Union[bytes, bytearray, mmap.mmap, MemoryBuffer]
```

Comment:
I think it can be just deleted as we don't have any references in code and have that alias already defined in .pyi file.
```suggestion
```

Reply:
Ok, they are referenced, nvm